### PR TITLE
ELEMENTS-742: Select multiple users for in permission view

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

**Comment by afjusto** [comment]
:+1:

File: nuxeo-document-permissions/nuxeo-popup-permission.html [hunk 1]
```diff
@@ -231,11 +232,11 @@ <h2>[[_computedTitle]]</h2>
         }
 
         _doSend(togglePopup) {
-          if (this.shareWithExternal &&
-            ((!this.updatingACE && !this.params.email) || (this.updatingACE && !this.params.username))) {
+          if (this.shareWithExternal
+        && ((!this.updatingACE && !this.params.email) || (this.updatingACE && !this.params.username))) {
```
Comment (Gabez0r) [inline nuxeo-document-permissions/nuxeo-popup-permission.html]: This line should have at least that same indentation of the if:
```
  if (this.shareWithExternal &&
     ((!this.updatingACE && ...
```

**Comment by Gabez0r** [comment]
👍

File: nuxeo-document-permissions/nuxeo-popup-permission.html [hunk 2]
```diff
@@ -78,7 +78,8 @@ <h2>[[_computedTitle]]</h2>
         <div hidden$="{{updatingACE}}">
           <div hidden$="{{shareWithExternal}}">
             <nuxeo-user-suggestion name="userGroup" label="[[i18n('popupPermission.userGroup')]]"
-                                   value="{{params.username}}"
+                                   value="{{params.users}}"
+                                   multiple="true"
```
Comment (guirenard) [inline nuxeo-document-permissions/nuxeo-popup-permission.html]: simply `multiple`

File: nuxeo-document-permissions/nuxeo-popup-permission.html [hunk 3]
```diff
@@ -284,32 +282,17 @@ <h2>[[_computedTitle]]</h2>
...
-            end: '',
-            notify: true,
-            comment: '',
-          };
+          const params = this._getResetParams();
 
           // force datebased and notification for external share
           if (shareWithExternal) {
-            this.set('selectedTimeFrame', 'datebased');
+            this.selectedTimeFrame = 'datebased';
```
Comment (guirenard) [inline nuxeo-document-permissions/nuxeo-popup-permission.html]: Aren't you sure the `set` wasn't used to notify top-level elements?

File: nuxeo-document-permissions/nuxeo-popup-permission.html [hunk 4]
```diff
@@ -333,7 +316,7 @@ <h2>[[_computedTitle]]</h2>
             }
 
             if (dateBased) {
-              this.set('selectedTimeFrame', 'datebased');
+              this.selectedTimeFrame = 'datebased';
```
Comment (guirenard) [inline nuxeo-document-permissions/nuxeo-popup-permission.html]: idem

File: nuxeo-document-permissions/nuxeo-popup-permission.html (hunk 3 above)
Comment (nmpcunha) [reply nuxeo-document-permissions/nuxeo-popup-permission.html]: Yes, it didn't have any notitfy

**Comment by guirenard** [comment]
:+1:

**Pull request merged**